src/resources/js/v2/rsvp-ari.js

```diff
@@ -0,0 +1,623 @@
+
+/**
+ * Configures RSVP ARI Object in the Global Tribe variable
+ *
+ * @since  TBD
+ *
+ * @type {PlainObject}
+ */
+tribe.tickets.rsvp.ari = {};
+
+/**
+ * Initializes in a Strict env the code that manages the RSVP block.
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.tickets.rsvp.block
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+	const $window = $( window );
+
+	/**
+	 * Selectors used for configuration and setup
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.selectors = {
+		container: '.tribe-tickets__rsvp-wrapper',
+		rsvpForm: 'form[name~="tribe-tickets-rsvp-form-ari"]',
+		hiddenElement: '.tribe-common-a11y-hidden',
+		addGuestButton: '.tribe-tickets__rsvp-ar-quantity-input-number--plus',
+		removeGuestButton: '.tribe-tickets__rsvp-ar-quantity-input-number--minus',
+		guestList: '.tribe-tickets__rsvp-ar-guest-list',
+		guestListItem: '.tribe-tickets__rsvp-ar-guest-list-item',
+		guestListItemTemplate: '.tribe-tickets__rsvp-ar-guest-list-item-template',
+		guestListItemButton: '.tribe-tickets__rsvp-ar-guest-list-item-button',
+		guestListItemButtonInactive: '.tribe-tickets__rsvp-ar-guest-list-item-button--inactive',
+		guestListItemButtonIcon: '.tribe-tickets__rsvp-ar-guest-icon',
+		guestFormWrapper: '.tribe-tickets__rsvp-ar-form',
+		guestFormFields: '.tribe-tickets__rsvp-ar-form-guest',
+		guestFormFieldsError: '.tribe-tickets__form-message--error',
+		guestFormFieldsTitle: '.tribe-tickets__rsvp-ar-form-title',
+		guestFormFieldsTemplate: '.tribe-tickets__rsvp-ar-form-guest-template',
+		nextGuestButton: '.tribe-tickets__rsvp-form-button--next',
+		submitButton: '.tribe-tickets__rsvp-form-button--submit',
+	};
+
+	/**
+	 * Go to guest.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of the RSVP container.
+	 * @param {number} guestNumber The guest number we want to go to.
+	 *
+	 * @return {void}
+	 */
+	obj.goToGuest = function( $container, guestNumber ) {
+		const $guestFormWrapper = $container.find( obj.selectors.guestFormWrapper );
+		const $targetGuestForm = $guestFormWrapper.find( obj.selectors.guestFormFields + '[data-guest-number="' + guestNumber + '"]' );
+		const $guestListButtons = $container.find( obj.selectors.guestListItemButton );
+		const $guestListButtonsIcon = $container.find( obj.selectors.guestListItemButtonIcon );
+
+		// Set all forms as hidden.
+		$container.find( obj.selectors.guestFormFields ).addClass( obj.selectors.hiddenElement.className() );
+		$container.find( obj.selectors.guestFormFields ).attr( 'hidden', true );
+
+		// Show the selected guest.
+		obj.showElement( $targetGuestForm );
+		$targetGuestForm.removeAttr( 'hidden' );
+
+		// Set the classes for inactive.
+		$guestListButtons.addClass( obj.selectors.guestListItemButtonInactive.className() );
+		$guestListButtons.attr( 'aria-selected', false );
+		$guestListButtons.attr( 'tabindex', -1 );
```

Comment:
@juanfra  This is already merged, but I thought I'd comment about this so you can fix in another pass:

The use of `tabindex` is really only good when you have keyboard navigation in place (up/down or left/right arrows). As you do not have that implemented, a keyboard user will not be able to use the tab key to focus on the next tab, and they will also not be able to use keyboard to activate the next tab.

My suggestion would be to remove the use of `tabindex` on the tabs until you can implement a keyboard navigation. The keyboard nav, IMO, wouldn't be high priority, but this definitely works against accessibility if we don't take these out. The `tabindex` on the tabpanel should be fine.